Lombiq.HelpfulExtensions/Lombiq.HelpfulExtensions.csproj

```diff
@@ -51,4 +57,14 @@
     <PackageReference Include="Lombiq.HelpfulLibraries.OrchardCore" Version="5.1.1" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Utilities\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions\Lombiq.NodeJs.Extensions.csproj" />
```

Comment:
Move this project reference under the same condition as Helpful Libraries. (`'$(NuGetBuild)' != 'true'"`). See  https://github.com/Lombiq/Orchard-Chart.js/blob/dev/Lombiq.ChartJs/Lombiq.ChartJs.csproj#L41 for an example.

Reply:
This was not addressed. See the linked example, it should be the same. HL and Node Js project references should be under the same condition (`'$(NuGetBuild)' != 'true'"`): https://github.com/Lombiq/Orchard-Chart.js/blob/dev/Lombiq.ChartJs/Lombiq.ChartJs.csproj#L39-L42

Now it's under a separate condition and the condition is wrong it's `'$(NuGetBuild)' == 'true'"` instead of `'$(NuGetBuild)' != 'true'"`

See:
https://lombiq.atlassian.net/wiki/spaces/DEV/pages/35225644/Checklist+before+you+can+say+a+development+task+is+completed#:~:text=If%20the%20task%20is%20already%20after%20the%20first%20code%20review%20and%20now%20you%20want%20to%20set%20it%20do%20Done%20the%20second%20time%3A%20Did%20you%20address%20all%20comments%20the%20reviewer%20added%20to%20your%20code%3F%20I.e.%20did%20you%20apply%20all%20the%20changes%20requested%20or%20did%20you%20reply%20if%20you%20want%20to%20debate%20a%20remark%3F